flake.nix

```diff
@@ -12,6 +12,7 @@
       flake = false;
     };
     melange.url = "github:melange-re/melange";
+    melange.inputs.nixpkgs.follows = "nixpkgs";
```

Comment:
instead of reusing our own nixpkgs, would it be possible to just insert an overlay into melange's nixpkgs?

Reply:
If I understood you correctly, I don't think that's a good idea. Melange's nixpkgs are my nix overlays, which is what's causing the issue of having 2 OCaml compilers in the first place (clambda and flambda).

Reply:
Right but we can override your overlay? We don't need a fast melange since we only run it in trivial test projects.

Reply:
I guess if our goal is to speed up the build of melange this is ideal.

I'm just worried that if you decide to modify or overlay your nixpkgs somehow, melange won't build in our dev shell.

Anyway it's enough to keep an eye out for this for now.

Reply:
Alright, I removed the `follows` line. It doesn't impact the build: we just download an extra tarball of `nixpkgs`.